aad/spring-cloud-azure-starter-active-directory/web-client-access-resource-server/aad-web-application/src/main/java/com/azure/spring/sample/aad/security/AadConditionalAccessFilter.java

```diff
@@ -29,6 +32,10 @@
  * client will throw {@link WebClientResponseException}, web-application can handle this exception to challenge the
  * user.
  *
+ * <p>
+ * <b>NOTE:</b> This feature in grant type {@link Constants#ON_BEHALF_OF} is provided by {@link AadOboOAuth2AuthorizedClientProvider},
+ * which has been deprecated and replaced with {@link JwtBearerOAuth2AuthorizedClientProvider}, which no longer provides conditional access.
```

Comment:
This comment will confuse customer. You can choose one of the following options:

Option 1. Delete `conditional access` feature in this sample.
Option 2. Demonstrate how to enable `conditional access` feature in this sample.

My suggestion is option 1.